source/community/ovirt-counter.md

```diff
@@ -0,0 +1,19 @@
+---
+title: oVirt Counter
+authors: sandrobonazzola
+page_classes: community
+---
+
+# oVirt Counter
+
+Once upon a time there was the [Linux Counter Project](https://en.wikipedia.org/wiki/Linux_Counter).
+If you want your oVirt deployment listed here you can edit this page following the link on the bottom of this page.
+
+**IMPORTANT!** Please note oVirt project has no way to verify added data correspond to reality.
```

Comment:
Suggestions:
1. `oVirt project` --> `the oVirt project`
2. `added data` --> `that the added data` or `whether the added data`

Fix: `correspond` --> `corresponds`?

Reply:
Thanks, fixed.